### PR TITLE
pytorch frontend - pointwise ops ```pow```

### PR DESCRIPTION
**Pull request by primetimepaper**
Issue: pow #3458

### CODE REVIEW

**Comment by primetimepaper** [comment]
@LINBOTANG123 I don't see how the failing checks could be related to my code changes here, could you please confirm this? Thanks

**Comment by primetimepaper** [comment]
@simonetgordon does this look okay to merge? There are some tests failing but I don't think it's related to my code, please confirm. Thanks

**Comment by primetimepaper** [comment]
@MarShaikh @simonetgordon Requesting a PR review, thanks in advance

**Review by MarShaikh — CHANGES_REQUESTED** [review]
Everything looks good, just resolve the conflicts and I'll merge. Thanks!

**Comment by primetimepaper** [comment]
@MarShaikh I think it should be okay now, please can you confirm this? Thanks in advance.

File: ivy_tests/test_ivy/test_frontends/test_torch/test_pointwise_ops.py [hunk 1]
```diff
@@ -957,6 +957,48 @@ def test_torch_sqrt(
...
+        input_dtypes=input_dtype,
+        as_variable_flags=as_variable,
+        with_out=with_out,
+        num_positional_args=num_positional_args,
+        native_array_flags=native_array,
+        fw=fw,
+        frontend="torch",
+        fn_tree="pow",
+        input=np.asarray(x[0], dtype=input_dtype[0]),
+        other=np.asarray(x[1], dtype=input_dtype[1]),
```
Comment (MarShaikh) [inline ivy_tests/test_ivy/test_frontends/test_torch/test_pointwise_ops.py:997]: This frontend test is incorrect. Please check the arguments that are being to the function again! As `pow` doesn't have an `other` argument from your function definition.

**Review by MarShaikh — CHANGES_REQUESTED** [review]
After taking a deeper look into the testing code, it seems like you haven't passed the appropriate function to the `pow` function. One of the arguments being passed is incorrect. Please check my comment! Thanks!

File: ivy_tests/test_ivy/test_frontends/test_torch/test_pointwise_ops.py (hunk 1 above)
Comment (primetimepaper) [reply ivy_tests/test_ivy/test_frontends/test_torch/test_pointwise_ops.py]: You're right, I must have messed that up when resolving the merge conflicts, thank you for spotting that. Re-requesting review, thanks in advance

**Comment by primetimepaper** [comment]
@MarShaikh Please confirm we are good to merge

**Pull request closed without merging**